README.md

```diff
@@ -3,3 +3,127 @@
 [Hydra](http://www.hydra-cg.com/spec/latest/core/) is a machine readable description for APIs.
 Hydra Box extends the API description with links to the actual code, which provides the API.
 Hydra Box will use such an API description to start a server which provides the API and dynamically loads the required code for it.
+
+## Getting started
+
+A hydra-box API is essentially an [expressjs](https://expressjs.com) app where it is yet another middleware. Three main steps 
+are necessary to set it up:
+
+1. Create a Hydra `ApiDocumentation` graph
+1. Add terms to declare how specific operations are implemented
+1. Create an express application using hydra-box
+
+For the sake of this tutorial, let's assume that the entrypoint and base URI of all resources is `https://example.app/` and
+the terms will be within an `https://examle.app/api#` namespace.
+
+### Create a Hydra `ApiDocumentation` graph
+
+First step is to create the API Documentation which describes the types and operations which the API supports. 
+This is easiest done by hand-crafting a static RDF file. It can be serialized in any format supported by `rdf-ext`.
```

Comment:
Let's add a link:

    [`rdf-ext`](https://github.com/rdf-ext/rdf-ext)

for those unfamiliar with it :)